libraries/chain/include/eosio/chain/controller.hpp

```diff
@@ -83,11 +83,13 @@ namespace eosio { namespace chain {
             uint16_t                 max_retained_block_files   = chain::config::default_max_retained_block_files;
             uint64_t                 blocks_log_stride          = chain::config::default_blocks_log_stride;
             backing_store_type       backing_store              = backing_store_type::CHAINBASE;
+            // rocksdb related. currently unused
```

Comment:
Remove all these and all the `default_` ones in `chain/config.hpp`

Reply:
Those things are saved in state database. So this means we will need to delete state and do replay on next bullish release. are we ok with that?

Reply:
They are not stored in the state database.

Reply:
good point. only `backing_store`. so I removed everything related to rocksdb from config but `backing_store`